src/promptflow-core/promptflow/executor/_prompty_executor.py

```diff
@@ -34,7 +35,7 @@ def _initialize_function(self):
         as executor input.
         """
         # If the function is not decorated with trace, add trace for it.
-        func = _traced(self.prompty)
+        func = _traced(self.prompty, trace_type=TraceType.FLOW)
```

Comment:
shall we add a new type for Promty? Is there any thing we can tell this is a prompty in trace UI?

Reply:
Add a new type for prompty.
![image](https://github.com/microsoft/promptflow/assets/17938940/cbe81f94-cc67-4c99-aabd-b8754fac0660)